app/controllers/tag_controller.rb

```diff
@@ -514,10 +514,12 @@ def stats
   end
 
   def comments
+    fetch_counts
```

Comment:
```suggestion
    @qids = Node.questions.where(status: 1).collect(&:nid)
    @wildcard = true if params[:id][-1..-1] == '*' # wildcard tags
    fetch_counts
    if params[:id].is_a? Integer
      @wiki = Node.find(params[:id])&.first
    elsif params[:id].to_s.match?(":")
      @wiki = Node.where(slug: params[:id].match('[^:]*$').to_s).try(:first)
    else
      @wiki = Node.where(path: "/wiki/#{params[:id]}").try(:first) || Node.where(path: "/#{params[:id]}").try(:first)
      @wiki = Node.where(slug: @wiki.power_tag('redirect'))&.first if @wiki&.has_power_tag('redirect') # use a redirected wiki page if it exists
    end
    @title = params[:id]
    @contributor_count = Tag.contributor_count(params[:id]) || 0
```

Reply:
Lines 520 - 527 were copied from https://github.com/publiclab/plots2/blob/9198f100b36f4d8bcbedf20b79112868e8ce1278/app/controllers/tag_controller.rb#L78-L85 so maybe we could take those lines of code and put it in it's own separate private method to make the card dry.